src/po/es.po

```diff
@@ -10219,3 +10199,12 @@ msgstr "nombre de la biblioteca dinámica MzScheme"
 msgid "name of the MzScheme GC dynamic library"
 msgstr "nombre de la biblioteca dinámica MzScheme GC"
 
+#, fuzzy, c-format
```

Comment:
that and the following lines can be removed.

Reply:
Yeah, I guess so. It was formatted by gettext and it must have tagged those lines and I didn't check after it. I'll delete those lines and check the other files.